stix_shifter_modules/crowdstrike_logscale/stix_transmission/error_mapper.py

```diff
@@ -0,0 +1,35 @@
+from stix_shifter_utils.utils.error_mapper_base import ErrorMapperBase
+from stix_shifter_utils.utils.error_response import ErrorCode
+from stix_shifter_utils.utils import logger
+
+error_mapping = {
+    100: ErrorCode.TRANSMISSION_QUERY_PARSING_ERROR,
```

Comment:
These error codes are not handled in connector level, since this has been already handled in STIX shifter level in stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py while making API calls.

Reply:
Yes. This would create Exception

Reply:
In that case, wouldn't it make sense to catch those exceptions? I don't think the default handler includes 500/429, so if the exceptions occurred after retrying 5 times or so, it would get thrown like any other exceptions.

Reply:
The  429 error code has been captured in RestApiClientAsync.py file in line 115 and exception is thrown from there. The other exception code 500 has now been added in error mapper file of connector.